### PR TITLE
Exclude multivalued vars from map markers overlay

### PR DESCRIPTION
**Pull request by chowington**
Fixes https://github.com/VEuPathDB/EdaNewIssues/issues/635

### CODE REVIEW

**Review by bobular — CHANGES_REQUESTED** [review]
Thanks. Works great but some streamlining recommended.

**Comment by chowington** [comment]
Thanks Bob! It was working for me, which is odd because I agree those changes needed to be made. Note that a few variable selections simply seem to not work at all, like "indicator of specimen collection date discontinuity" and "Geolocation provenance"

**Comment by bobular** [comment]
Indeed - I since noted that main is broken (my fault) until https://github.com/VEuPathDB/web-monorepo/pull/255 is merged.

I will test again right now and be careful with my choice of variables.

**Review by bobular — APPROVED** [review]
All seems good now, thanks!

The mouse-overs on the greyed out vars are not very helpful but I have opened another issue for this, as it requires some discussion and is not in scope here
https://github.com/VEuPathDB/EdaNewIssues/issues/644

File: packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/PieMarkerConfigurationMenu.tsx [hunk 1]
```diff
@@ -65,12 +66,15 @@ export function PieMarkerConfigurationMenu({
       </p>
       <InputVariables
         showClearSelectionButton={false}
-        inputs={[{ name: 'overlay', label: 'Variable', titleOverride: ' ' }]}
+        inputs={[
+          { name: 'overlayVariable', label: 'Variable', titleOverride: ' ' },
+        ]}
         entities={entities}
         selectedVariables={{ overlay: configuration.selectedVariable }}
```
[the remark this answers is not in the document]
Comment (dmfalke) [reply packages/libs/eda/src/lib/map/analysis/MarkerConfiguration/PieMarkerConfigurationMenu.tsx]: Yep, `inputs[].name` should match up with keys of `selectedVariables`.

**Pull request merged**